Heart/Message/Presentation/Controllers/MessagesController.php

```diff
@@ -4,17 +4,29 @@
 
 use App\Http\Controllers\Controller;
 use Heart\Message\Application\NewMessage;
+use Heart\Message\Application\NewVoiceMessage;
 use Heart\Message\Presentation\Request\CreateMessageRequest;
+use Heart\Message\Presentation\Request\CreateVoiceMessageRequest;
 use Illuminate\Http\Response;
 
 class MessagesController extends Controller
 {
     public function postMessage(
         CreateMessageRequest $request,
         string $provider,
-        NewMessage $action,
+        NewMessage $newMessage,
     ): Response {
-        $action->handle($request->validated());
+        $newMessage->persist($request->validated());
```

Comment:
It would be nice to have a PostMessageViewModel to represent the received message.
Again, I know we don't have time now, this is just a reminder to latter.